aws-ecs-byovpc/error-destroy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -u
+set -o pipefail
+
+echo "executing error-destroy script"
+
+echo "ensuring AWS is setup"
+aws sts get-caller-identity > /dev/null
+
+echo "looking for ENIs which were orphaned by vpc-cni plugin"
```

Comment:
I thought this was just for EKS?

Reply:
@jordan-acosta leaving this cleanup in here, for now so I can monitor from a known good, and then if we see on deprovision that no ENIs are leaking, we can go ahead and remove this.